glidertools/utils.py

```diff
@@ -178,6 +178,8 @@ def calc_dive_phase(time, depth):
         glider time dimension
     depth : np.array [float]
         depth (m) or pressure (dbar) if depth not avail
+    dive_depth_threshold : int
```

Comment:
Why is this an int? Should't we allow depths like `10.5`?

Reply:
yep, I'll change that.